### PR TITLE
test(cardano-services): remove testnet snapshot from rewards, stake pool and CLI tests

### PR DESCRIPTION
**Pull request by AngelCastilloB**
# Context

Final step of the work needed to completely remove the old testnet snapshot from the cardano-services tests.

# Important Changes Introduced

+ Updated the snapshot to include pool retired, retiring, active and activating both with and without pledge.
+ Removed snapshot from rewards tests.
+ Removed snapshot from stake pool tests.
+ Removed snapshot from cli.

### CODE REVIEW

File: packages/cardano-services/test/NetworkInfo/fixtures/FixtureBuilder.ts [hunk 1]
```diff
@@ -19,4 +19,14 @@ export class NetworkInfoFixtureBuilder {
 
     return result.rows.map(({ max_supply }) => BigInt(max_supply))[0];
   }
+
+  public async getLasKnownEpoch(): Promise<number> {
+    this.#logger.debug('About to fetch las known epoch');
```
Comment (rhyslbw) [inline packages/cardano-services/test/NetworkInfo/fixtures/FixtureBuilder.ts:24]:
```suggestion
    this.#logger.debug('About to fetch last known epoch');
```

**Review by rhyslbw — CHANGES_REQUESTED** [review]
Looking good @AngelCastilloB, just some clarifications and suggestion related to the commits.

1. Are the changes in 33ada8da3489ff02f40abf7bbade4d1f1ea8860d required for a26b80dc04c88c6f377e6b79da273ba865ea3ec1? If so, best to squash into one.
2. 1ea94bb7a76a3391af59fca550f032241221a5fc would be best to reorder as the first commit since it's a fix from a past refactor, and it looks like you needed for prior commits to have passing tests (please correct me if I'm wrong here 😄 )
3. Can 33ada8da3489ff02f40abf7bbade4d1f1ea8860d and 1e38a27fb29135422afcae23a069aa9b509ff6df be considered a single atomic change? 
4. 2a68b6f64c0a3b24571a601ed74c949938630817 `test: removed snapshot from CLI tests` would be better described as `test: refactor CLI tests to use dynamic fixtures`
5. f13faec2d5c2091e961e95d1f5fb509c4e25b6d0 `test(cardano-services): finish removing snapshot from network info tests` would be better described as `test(cardano-services): finish refactoring network info tests to use dynamic fixtures`
6. d738f94e2629f66331682a9b11a8c77c9ed2e38b `test(cardano-services): removed snapshot from polling tests` - `test(cardano-services): refactor poll service test to use dynamic fixtures`
7. a519dfd00b355210c58ae86ead45a2c5cd1e6f89 can be squashed into d738f94e2629f66331682a9b11a8c77c9ed2e38b, as that's the last dependency on the _testnet_ snapshot.

**Review by rhyslbw — APPROVED** [review]
Great work @AngelCastilloB :european_castle:

File: packages/cardano-services/test/jest-setup/rebuild-test-db.sh [hunk 2]
```diff
@@ -7,9 +7,11 @@ WORKSPACE_ROOT="$(dirname "${PACKAGES_DIR}")"
 SECRETS_DIR="$PACKAGES_DIR"/cardano-services/placeholder-secrets
 
 yarn --cwd "$PACKAGES_DIR"/e2e local-network:down
-yarn --cwd "$PACKAGES_DIR"/e2e local-network:up -d
+yarn --cwd "$PACKAGES_DIR"/e2e local-network:up -d --build
 yarn --cwd "$WORKSPACE_ROOT" build
 yarn --cwd "$PACKAGES_DIR"/e2e test:wallet
+yarn --cwd "$PACKAGES_DIR"/e2e test:long-running --forceExit
```
Comment (mkazlauskas) [inline packages/cardano-services/test/jest-setup/rebuild-test-db.sh:13]: This is no longer necessary as we [added --forceExit to package.json script](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/e2e/package.json#L20)

**Review by mkazlauskas — CHANGES_REQUESTED** [review]
Nice work! Just 1 minor change request

**Review by rhyslbw — APPROVED** [review]

**Review by mkazlauskas — APPROVED** [review]

**Pull request merged**